src/consensus/pbft_manager.cpp

```diff
@@ -635,7 +640,8 @@ void PbftManager::loopBackFinishState_() {
   db_->commitWriteBatch(batch);
   next_voted_soft_value_ = false;
   next_voted_null_block_hash_ = false;
-  next_step_time_ms_ = step_ * LAMBDA_ms;
+  assert(step_ >= startingStepInRound_);
+  next_step_time_ms_ = (1 + step_ - startingStepInRound_) * LAMBDA_ms;
```

Comment:
Does step_ always equal to startingStepInRound_? If yes, next_step_time_ms_ always is LAMBDA_ms time